Sources/PaymentsCore/Networking/APIClientError.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+enum APIClientError {
```

Comment:
There is no need for this to be public! This file is a way for us internally to consolidate all error creation per module / domain. Having them all in one place helps us make sure we're consistent in our instantiation, wording, etc per error.

But this file doesn't need to be public since a merchant will get back the relevant `PayPalSDKError` in the completion of whatever checkout flow they kicked off (ex: `cardClient.approveOrder(orderID: String, card: Card, completion: @escaping (Result<OrderData, PayPalSDKError>) -> Void)`)